third_party/move/tools/move-package/Cargo.toml

```diff
@@ -46,6 +46,7 @@ whoami = { workspace = true }
 [dev-dependencies]
 datatest-stable = { workspace = true }
 move-stdlib = { path = "../../move-stdlib" }
```

Comment:
third_party/ should not refer to aptos-move at this point.  Move this test somewhere under aptos-move/

Reply:
It's in [dev-dependencies] and only used for testing. Is this ok?

Reply:
No.  Add a test somewhere under aptos-move if you need to use Aptos libs.  Don't create a dep cycle.

You need to become familiar with that tree anyway.  Search for Move.toml files under "tests" dirs to find some examples.

Reply:
I can move the test.

Can you explain? I think development dependencies are created exactly to avoid dep cycles.

Reply:
We may have some partners who use our third_party tree without the Aptos stuff.  We are still trying to keep it independent.  Although that may be broken at some point, this is not enough reason to do it.